src/Microsoft.AspNet.Http.Features/IRequestCookies.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNet.Http
+{
+    /// <summary>
+    /// Represents the HttpRequest cookie collection
+    /// </summary>
+    public interface IRequestCookies : IDictionary<string, StringValues>
```

Comment:
There was a build order problem; but I've moved `IFormCollection`/`IFormFile`/`IFormFileCollection`/`IQueryCollection`/`IRequestCookies` back to Abstractions and it doesn't seem to be an issue any more... Will update PR.